CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+## How to contribute ##
+
+The following are the guidelines, everyone should use to contribute to utPLSQL.  
+Changes are welcome from all members of the Community. 
+
+## Getting Started ##
+
+1. Create a [GitHub Account](https://github.com/join).
+2. Fork the utPLSQLRepository and setup your local Repository.
+     * Each of of the steps below are detailed [How to Fork](https://help.github.com/articles/fork-a-repo) article!
+     * Clone your Fork to your local machine.
+     * Configure "upstream" remote to the [master utPLSQL repository](https://github.com/utPLSQL/utPLSQL.git).
+3. For each change you want to make:       
+     * Create a new branch for your change. 
+     * Make your change in your new branch. 
+         * Although changes can be made in the master branch, it easier long term if a new branch is used.
+     * **Verify code compiles and unit tests still pass.** 
+         * The quickest way to have a Pull Request not be accepted, is to submit code that does not compile or pass tests.
+     * Commit change to your local repository.
+     * Push change to your remote repository
+     * Submit a [Pull Request](https://help.github.com/articles/using-pull-requests).
+     * Note: local and remote branches can be deleted after pull request has been accepted.
+
+**Note:** Getting changes from others requires [Syncing your Local repository](https://help.github.com/articles/syncing-a-fork) with Master utPLSQL repository.    This can happen at any time.
+
+
+## Dependencies ##
+
+* Oracle Version XX.X or Later
+
+## New to GIT ##
+
+If you are new to GIT here are some links to help you with understanding how it works.    
+
+- [GIT Documentation](http://git-scm.com/doc)
+- [Atlassian Git Tutorial](https://www.atlassian.com/git/tutorial/git-basics)
+- [What are other resources for learning Git and GitHub](https://help.github.com/articles/what-are-other-good-resources-for-learning-git-and-github) 
+
+
+## Feedback and Collaboration ##
+
+Find out where to put this, other projects have used Google+ Communities.
```

Comment:
Apart from Sourceforge (which we're moving away from), we don't really have anywhere else. In the interest of keeping things simple, I was hoping to keep as much as possible in one place. I don't have much experience with GitHub yet - do you think we can just use GitHub, or should we consider something else as well? Is there anything that integrates nicely with GitHub?